cloud/amazon/ec2_elb_lb.py

```diff
@@ -107,6 +107,12 @@
     required: false
     aliases: []
     version_added: "1.8"
+  idle_timeout:
+     description:
+       - Sets the number of seconds a connection can be idle before the load balancer terminates the connection
+     required: false
+     aliases: []
```

Comment:
@digital-abyss Yes. If you look at the link provided by @gregdek, the module checklist calls these out as not being necessary when there are no aliases. I know the existing code had them. However, it is my opinion that new code should not ignore the guidelines just to be consistent with the old code. We will never have code that passes the checklist if we don't reverse our old habits as we add new code.